services/user/Identity/service/src/tests/helpers/test_helpers.rs

```diff
@@ -1,128 +1,170 @@
+use super::gql::Queryable;
 use crate::service::{Attestation, AttestationStats};
-use crate::SERVICE;
+use crate::Wrapper as Service;
+use psibase::services::http_server;
 use psibase::AccountNumber;
+use std::fmt::Debug;
+
+pub trait HasQueryFields {
+    const QUERY_FIELDS: &'static str;
+}
```

Comment:
Yeah a lot of this was just timeboxed because it's only in the identity service, but could serve as a foundation for future generalizing if we end up moving some into the tester lib.